approval_polls/templates/approval_polls/embed_instructions.html

```diff
@@ -1,7 +1,43 @@
 {% extends 'base.html' %}
 
 {% block content %}
-  Put this code in your site:
-  <textarea rows=3 cols=100><iframe src='{{ link }}' width='250'></iframe></textarea>
+
+<script type='text/javascript'>
+
+// thanks to http://www.dyn-web.com
+function getDocHeight(doc) {
+    doc = doc || document;
+    var body = doc.body, html = doc.documentElement;
+    var height = Math.max( body.scrollHeight, body.offsetHeight, 
```

Comment:
It will be better to use libraries like jQuery/Dojo/Google Closure to do DOM manipulation (like setting/getting properties of nodes) as these libraries are tried and tested and support most popular browsers. This way, the code will also be more concise.
For example to get the height of the body using jQuery would just be $('body').height()